### PR TITLE
watch was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/_watch.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    left: 50%;
+    top: 15%;
+    transform: translateX(-50%);
+    z-index: -1;
+    background-color: gray;
+    animation: rotate 3600s linear infinite normal;
+    transform-origin: bottom center;
+  }
+
+  @keyframes rotate {
```
Comment (OlEzhA0) [inline src/styles/blocks/_watch.scss:42]: move keyframe to separate file

**Review by OlEzhA0 — APPROVED** [review]
check your UI)
Screenshot 2023-02-20 at 11 38 37 AM
hey! GJ

File: src/styles/blocks/_watch.scss (hunk 1 above)
Comment (BilokonYaroslav) [reply src/styles/blocks/_watch.scss]: What You mean "check your UI)"?

**Review by OleziO — CHANGES_REQUESTED** [review]
The red clock hand is the second hand, and the blue hand is the minute hand. Fix it

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**